need to correct the macros

As title, obvious the code there is wrong.
```
--- a/plugins/vst_base/RemoteVstPlugin.cpp	2016-12-10 03:23:56.320741067 +0000
+++ b/plugins/vst_base/RemoteVstPlugin.cpp	2016-12-10 03:23:56.320741067 +0000
@@ -59,13 +59,11 @@
 #define USE_WS_PREFIX
 #include <windows.h>
 
-#ifdef LMMS_BUILD_WIN32
-#ifdef LMMS_BUILD_WIN64
+#if defined(LMMS_BUILD_WIN32) || defined(LMMS_BUILD_WIN64)
 #include "basename.c"
 #else
 #include <libgen.h>
 #endif
-#endif
 
 
 #include <vector>
```

And if you don't mind, please add this, linux != glibc:
```
--- a/include/lmms_math.h	2016-12-10 03:20:33.262213993 +0000
+++ b/include/lmms_math.h	2016-12-10 03:20:33.262213993 +0000
@@ -34,7 +34,7 @@
 #include <cmath>
 using namespace std;
 
-#if defined (LMMS_BUILD_WIN32) || defined (LMMS_BUILD_APPLE) || defined(LMMS_BUILD_HAIKU)  || defined (__FreeBSD__) || defined(__OpenBSD__)
+#if defined (LMMS_BUILD_WIN32) || defined (LMMS_BUILD_APPLE) || defined(LMMS_BUILD_HAIKU)  || defined (__FreeBSD__) || defined(__OpenBSD__) || !defined(__GLIBC__)
 #ifndef isnanf
 #define isnanf(x)	isnan(x)
 #endif
```

Even with glibc, there's warning, if we add a error-as-wrong flag to gcc, we get a broken build then:
```
s.c: In function 'main':
s.c:5:3: warning: implicit declaration of function 'basename' [-Wimplicit-function-declaration]
   basename("ss");
```

--- a/plugins/vst_base/RemoteVstPlugin.cpp
+++ b/plugins/vst_base/RemoteVstPlugin.cpp
@@ -59,12 +59,10 @@
 #define USE_WS_PREFIX
 #include <windows.h>
 
-#ifdef LMMS_BUILD_WIN32
-#ifdef LMMS_BUILD_WIN64
+#if defined(LMMS_BUILD_WIN32) || defined(LMMS_BUILD_WIN64)
 #include "basename.c"
 #else
 #include <libgen.h>
-#endif
 #endif
 
 

